csharp/src/Microsoft.ML.OnnxRuntime/SessionOptions.cs

```diff
@@ -47,6 +60,7 @@ public SessionOptions()
             :base(IntPtr.Zero, true)
         {
             NativeApiStatus.VerifySuccess(NativeMethods.OrtCreateSessionOptions(out handle));
+            NativeApiStatus.VerifySuccess(NativeMethods.OrtSetSessionProgrammingProjection(handle, OrtProgrammingProjection.ORT_PROJECTION_CROSSPLAT_CSHARP));
```

Comment:
what happens if someone creates a session without using sessionoptions?

Reply:
It will create a default sessionOptions when we'll set the projection language

Reply:
> > This doesn't look to me like a per session concept. I don't think the projection will change from session to session. Who is the user of this API? Is it the projection (language) binding developers?
> 
> Yes, the user of this API are projection developers.